lib/crypto/packaged.py

```diff
@@ -27,7 +27,8 @@ class SigningError(Exception):
 def get_endpoint(file_obj):
     """Get the endpoint to sign the file, depending on its review status."""
     server = settings.SIGNING_SERVER
-    if file_obj.version.addon.status != amo.STATUS_PUBLIC:
+    if (file_obj.status == amo.STATUS_BETA or
+            file_obj.version.addon.status != amo.STATUS_PUBLIC):
```

Comment:
One is checking the file status, the other is checking the addon status. If the add-on isn't PUBLIC or the file is BETA, we sign with prelim. Right?